antlr4/PostgreSQLParser.g4

```diff
@@ -1198,15 +1210,20 @@ show_stmt
     ;
 
 truncate_stmt
-    : todo_implement
+    : TRUNCATE TABLE? ONLY? table_name_ STAR? (COMMA ONLY? identifier_list)?
```

Comment:
I think the second `ONLY?` can be removed here.

Reply:
See [src/test/resources/sql/truncate/a31f29bb.sql](https://github.com/tshprecher/antlr_psql/blob/master/src/test/resources/sql/truncate/a31f29bb.sql)
```TRUNCATE ONLY trunc_fb, ONLY trunc_fa```